ql/src/java/org/apache/hadoop/hive/ql/ddl/table/creation/createlike/CreateTableLikeDesc.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.hadoop.hive.ql.ddl.table.creation;
```

Comment:
Instead of `org.apache.hadoop.hive.ql.ddl.table.creation.create` or `org.apache.hadoop.hive.ql.ddl.table.creation.drop`, isn't it more natural to have `org.apache.hadoop.hive.ql.ddl.table.create` or `org.apache.hadoop.hive.ql.ddl.table.drop`?

Reply:
I choose this package structure was to have a structure for the table related operations. There are 46 of them, and having a package for each of them would mean an endless list of packages, this is why they were put into packages by categories.

Yet I understand your point, if someone is looking at an import it seems odd. One way to solve is that you suggested, or we could replace the "creation" with something else, like "basic", or "existence", or if you have a better suggestion :) This way we'll have a more balanced structure, where there is the main category (table), the sub category (some label), and then the actual command. But if you believe that having the undoubtedly most straightforward name that you've suggested is more important than the balanced package structure, I'm totally fine with that too, I think the most important thing is to break up the DDLSemanticAnalyzer here, and this question is secondary.

Please let me know what you think.

Reply:
Yes, I believe having `create` or `drop` packages within `table` package, even if we have them next to `column`, `constraint`, etc., is more intuitive. It seems the additional subpackage does not provide clarity, e.g., `creation.drop` or `creation.create` is confusing. Also the `creation` package seem small enough already, hence another option is that these classes go directly to the `table` package. I believe there are many options, right now it is a matter of taste... I would just avoid redundant packages.

Reply:
Ok, I've updated the package structure, now the 4 operations are at

`org.apache.hadoop.hive.ql.ddl.table.create`
`org.apache.hadoop.hive.ql.ddl.table.create.like`
`org.apache.hadoop.hive.ql.ddl.table.create.show`
`org.apache.hadoop.hive.ql.ddl.table.drop`